shared/actions/chat2/index.js

```diff
@@ -1376,7 +1376,8 @@ const _maybeAutoselectNewestConversation = (
     action.payload.conversationIDKey === selected
   ) {
     // Intentional fall-through -- force select a new one
-  } else if (selected !== Constants.noConversationIDKey) {
+  } else if (selected !== Constants.noConversationIDKey && selected !== Constants.pendingConversationIDKey) {
```

Comment:
maybe use this: https://github.com/keybase/client/blob/master/shared/constants/types/chat2/common.js#L23